docs/fsharp/index.md

```diff
@@ -44,15 +44,23 @@ To learn more and get involved, check out [fsharp.org](http://fsharp.org).
 
 ## Documentation
 
-* [Getting Started](tutorials/getting-started/index.md)
-* [Introduction to Functional Programming](introduction-to-functional-programming/index.md)
 * [Tutorials](tutorials/index.md)
+* [Introduction to Functional Programming](introduction-to-functional-programming/index.md)
 * [Language Reference](language-reference/index.md)
 * [F# Core Library Reference](https://msdn.microsoft.com/en-us/visualfsharpdocs/conceptual/fsharp-core-library-reference)
 
+## Online Learning Resources
```

Comment:
Yeah, I was looking for a way to differentiate between reading material and video material, since people often prefer one over the other.  Perhaps something like this?

## Further Learning Resources
### Online Reading Material
### Video Resources